packages/paste-website/src/pages/components/checkbox/index.mdx

```diff
@@ -145,7 +160,7 @@ Multiple checkboxes and their labels are grouped together with a common group co
 
 ### Checkbox group with help text
 
-You can provide additional information about the group with the use of help text. Help text can appear after the group label but before the first group member. Each item in the group may also provide their own, individual help text.
+Use help text to give more context about a Checkbox group. In order to maintain styling consistency, be sure to use the helpText prop here instead of using the Help Text component. Individual checkboxes can also include their own help text if needed.
```

Comment:
```suggestion
Use help text to give more context about a Checkbox group. In order to maintain styling consistency, be sure to use the `helpText` prop here instead of using the Help Text component. Individual checkboxes can also include their own help text if needed.
```